projects/ion/src/lib/popover/component/popover.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, Input } from '@angular/core';
+import { Subject } from 'rxjs';
+import { SafeAny } from '../../utils/safe-any';
+
+import { PopoverPosition } from '../../core/types/popover';
+import { IconType, IonButtonProps } from '../../core/types';
+
+export interface PopoverProps {
+  ionPopoverTitle: string;
+  ionPopoverBody: SafeAny;
+  ionPopoverActions?: IonButtonProps[];
+  ionPopoverIcon?: IconType;
+  ionPopoverIconClose?: boolean;
+  ionPopoverPosition?: PopoverPosition;
+}
+
+@Component({
+  selector: 'ion-popover',
+  templateUrl: './popover.component.html',
+  styleUrls: ['./popover.component.scss'],
+  exportAs: 'PopoverComponent',
+})
+export class IonPopoverComponent {
+  @Input() ionPopoverTitle: string;
+  @Input() ionPopoverBody: SafeAny;
+  @Input() ionPopoverActions?: IonButtonProps[];
+  @Input() ionPopoverIcon?: IconType;
+  @Input() ionPopoverIconClose?: boolean = false;
```

Comment:
```suggestion
  @Input() ionPopoverIconClose = false;
```
Type assertions aren't necessary when you're setting a primitive value in the declaration.